src/collection-preferences/utils.tsx

```diff
@@ -46,17 +47,17 @@ export const ModalContentLayout = ({ left, right }: ModalContentLayoutProps) =>
   if (smallContainer) {
     return (
       <div ref={ref}>
-        <div>{left}</div>
-        {right && <div className={styles['second-column-small']}>{right}</div>}
+        {left && <div>{left}</div>}
+        {right && <div className={clsx(left && styles['second-column-small'])}>{right}</div>}
```

Comment:
Minor: I don't think we gain anything from using clsx here if we're not combining class names. 

So maybe readability vs. unnecessary use of a library to avoid a ternary operator?
```
className={left ? styles['second-column-small'] : null}
```

I don't think I have a strong preference since its not a new dependency and we use it elsewhere.

Reply:
Yeah I chose those for better readability. A ternary operator just seemed a little unusual here. And since we generally use `clsx` across the code base, I thought this was nicer.